### ISSUE TITLE
False positive for @pandacss/no-unsafe-token-fn-usage

### ISSUE THREAD

**Issue by yangchristina** [chakra-ui/eslint-plugin-panda#198]
This is an example of correct code in the docs:
```tsx
import { Circle } from './panda/jsx';

function App(){
  return <Circle _hover={{  border: 'solid 1px {colors.blue.400}' }} />;
}
```
However, this fails inside `css`:
<img width="861" alt="Screenshot 2024-12-26 at 7 49 47 PM" src="https://github.com/user-attachments/assets/47b4e3aa-bf9a-475a-8932-2cbfe5bc4ac7" />

But splitting the token value out fixes the lint warning:
![image](https://github.com/user-attachments/assets/994ea8a7-c876-45bc-904e-2ac8616168ad)

I've noticed this seems to work fine in config recipes + `cva`.

However, this line below case does cause a lint warning in `cva` for  @pandacss/no-unsafe-token-fn-usage:
`transition: transform {durations.veryFast} ease-out, fill-opacity {durations.medium} ease-out`
I wasn't sure if this was intended.

## Reproduce at:
https://github.com/cybersemics/em/pull/2739 by commenting out `'@pandacss/no-unsafe-token-fn-usage': 'off',`
running `yarn lint` gives various warnings with `@pandacss/no-unsafe-token-fn-usage: "warning"`
The example given above is from `src/components/DropDownMenu.tsx`

**Comment by anubra266** [comment]
Thanks for opening this @yangchristina 
Now fixed in [v0.2.3](https://github.com/chakra-ui/eslint-plugin-panda/releases/tag/%40pandacss%2Feslint-plugin%400.2.3)
